x-pack/plugins/actions/README.md

```diff
@@ -272,6 +242,160 @@ const result = await actionsClient.execute({
+| -------- | -------------------------------- | ------ |
+| id       | The id of the issue type in Jira | string |
+
+#### `subActionParams (issues)`
+
+| Property | Description             | Type   |
+| -------- | ----------------------- | ------ |
+| title    | The title to search for | string |
+
+#### `subActionParams (issue)`
+
+| Property | Description                 | Type   |
+| -------- | --------------------------- | ------ |
+| id       | The id of the issue in Jira | string |
+
+#### `subActionParams (getFields)`
+
+No parameters for `getFields` sub-action. Provide an empty object `{}`.
+
+---
+## IBM Resilient
+
+The [IBM Resilient user documentation `params`](https://www.elastic.co/guide/en/kibana/master/resilient-action-type.html) lists configuration properties for the `pushToService` subaction. In addition, several other subaction types are available.
+
+### `params`
+
+| Property        | Description                                                                                        | Type   |
+| --------------- | -------------------------------------------------------------------------------------------------- | ------ |
+| subAction       | The sub action to perform. It can be `pushToService`, `getFields`, `incidentTypes`, and `severity` | string |
+| subActionParams | The parameters of the sub action                                                                   | object |
+
+#### `subActionParams (pushToService)`
+
+| Property | Description                                                                                                   | Type                  |
+| -------- | ------------------------------------------------------------------------------------------------------------- | --------------------- |
+| incident | The IBM Resilient incident.                                                                                   | object                |
+| comments | The comments of the case. A comment is of the form `{ commentId: string, version: string, comment: string }`. | object[] _(optional)_ |
+
+The following table describes the properties of the `incident` object.
+
+| Property      | Description                                                                                                                  | Type                  |
+| ------------- | ---------------------------------------------------------------------------------------------------------------------------- | --------------------- |
+| name          | The title of the incident                                                                                                    | string _(optional)_   |
+| description   | The description of the incident                                                                                              | string _(optional)_   |
+| externalId    | The id of the incident in IBM Resilient. If presented the incident will be update. Otherwise a new incident will be created. | string _(optional)_   |
```

Comment:
```suggestion
| externalId    | The ID of the incident in IBM Resilient. If present, the incident is updated. Otherwise, a new incident is created. | string _(optional)_   |
```